pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -321,10 +322,11 @@ stdenv.mkDerivation rec {
     '';
 
     genericPatches = ''
-      # Substitute python's stub shebang to plain python path. (see TODO add pr URL)
+      # Substitute j2objc and objc wrapper's python shebang to plain python path.
       # See also `postFixup` where python is added to $out/nix-support
```

Comment:
both python2 and 3 are added to $out/nix-support

Reply:
I've made the comments more precise about the Python versions.